crates/tools/riddle/src/rust/mod.rs

```diff
@@ -30,6 +30,7 @@ pub fn from_reader(
     gen.std = config.remove("STD").is_some();
     gen.sys = gen.std || config.remove("SYS").is_some();
     gen.implement = config.remove("IMPLEMENT").is_some();
+    gen.minimal = config.remove("MINIMAL").is_some();
```

Comment:
Nit: I'd prefer to see `--minimal` etc. instead of `-config MINIMAL`

Reply:
Yeah, I'm still mulling this over. The `-config` options are just passed onto the respective generators (idl/rust/winmd) and will tend to be very specific to one or the other, which is why I didn't make them first-class options.